config-sources/zookeeper/example/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
```

Comment:
Given there's already a test, i'm not sure about an example here as well.

My reason for that is it's an additional thing to keep up to date, and could cause issues with changes in smallrye-config being required for the example to work but WF Swarm/Thorntail not having a version available with the necessary smallrye-config changes. We end up in a bit of a loop